content/reference/endpoints/booking/discount_codes.md

```diff
@@ -0,0 +1,116 @@
+minimum_spend            | Float                            | Read       | Minimum amount that must be spent in order to apply Discount Code.
+start_date               | [Date](/reference/enums#formats) | Read       | The start date from which the Discount Code will be applicable.
+end_date                 | [Date](/reference/enums#formats) | Read       | The expiration date of the Discount Code.
+-------------------------|----------------------------------|------------|---------------------------------------------
+created_at               | [Time](/reference/enums#formats) | Read       | Discount Code create time
+updated_at               | [Time](/reference/enums#formats) | Read       | Discount Code update time
+canceled_at              | [Time](/reference/enums#formats) | Read       | Discount Code soft delete time
+{: class="table table-bordered"}
+
+</div>
+<div class="tab-pane" id="discount_code_write" markdown="1">
+
+Name                     | Type                                                          | Read/Write | Description
+-------------------------|---------------------------------------------------------------|------------|----------------
+id                       | String[UUID]                                                  | Read       | Discount Code's id.
+discount_code            | String                                                        | Read/Write | Discount Code itself.
+currency                 | String                                                        | Read/Write | Discount Code's currency.
+fixed_amount             | Float                                                         | Read/Write | Fixed amount of discount.
+percentage               | integer                                                       | Read/Write | Discount percentage.
+all_rentals              | Boolean                                                       | Read       | Flag, that indicates that this discount is applicable to all rentals with the same currency.
+number_of_times          | Integer                                                       | Read/Write | Number of times the Discount Code can be applied.
+minimum_nights_booked    | integer                                                       | Read/Write | Minimum number of nights that must be booked in order to apply Discount Code.
+minimum_spend            | Float                                                         | Read/Write | Minimum amount that must be spent in order to apply Discount Code.
+start_date               | [Date](/reference/enums#formats)                              | Read/Write | The start date from which the Discount Code will be applicable.
+end_date                 | [Date](/reference/enums#formats)                              | Read/Write | The expiration date of the Discount Code.
+rentals_eligibility      | [String](/reference/enums#discount-code-rentals-eligibility)  | Write      | What rentals this Discount Code can be applied to.
```

Comment:
good to mention here that this should remain empty if all_rentals is true. Also, if all rentals is true - array is ignored.

Reply:
Added a note that `rental_ids` should be empty when `rentals_eligibility` is `"all_rentals"`